src/encoded/static/components/search.js

```diff
@@ -723,11 +723,31 @@ class Facet extends React.Component {
 
         // Bind `this` to non-React methods.
         this.handleClick = this.handleClick.bind(this);
+        this.scrollEvent = this.scrollEvent.bind(this);
     }
 
     handleClick() {
         this.setState(prevState => ({ facetOpen: !prevState.facetOpen }));
     }
+    
+    scrollEvent(e) {
```

Comment:
I’m getting a lot of ESLint/AirBnB errors in VSCode starting here and going on for many lines. They’ll all have to be cleaned up. In this particular case, `scrollEvent` has to be moved out to its own top-level function because it doesn’t refer to `this`, so it can’t exist in a class unless you make it a `static` method. But if you move it to its own function, then both Facet and TypeaheadFacet can share it.